spec/system/task_spec.rb

```diff
@@ -85,8 +78,7 @@
 
       it '既にステータスが完了のタスクのステータスを変更した場合、Taskの完了日が更新されないこと' do
         # TODO: FactoryBotのtraitを利用してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id, status: :done, completion_date: Time.current.yesterday)
+        task = create(:task, :done_project, project_id: project.id)
```

Comment:
この部分もassociationの設定により省略可能ですね。
```suggestion
        task = create(:task, :done_project)
```

Reply:
d0ddab3 で修正しました。